tasks/run-matlab-command/v0/task.json

```diff
@@ -20,6 +20,14 @@
             "required": true,
             "defaultValue": "",
             "helpMarkDown": "Script, function, or statement to execute. If the value of `command` is the name of a MATLAB script or function, do not specify the file extension. If you specify more than one MATLAB command, use a comma or semicolon to separate the commands."
+        },
+        {
+            "name": "startup-options",
```

Comment:
I think name is camelCase for this integration. See the run-matlab-tests/task.json file for consistency.

Reply:
Thanks! I totally missed that.